### PR TITLE
Styled  mobile app background

### PR DESCRIPTION
**Pull request by bludojr**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Simple Todo App</title>
+</head>
+
+<body class="s-lvh" style="background: #F9FAFB;">
```
Comment (amalisaah) [inline index.html:11]: this is fine but quite irregular you could have added the style to the style sheet 
css in the style sheet will overwrite this 

also what is the essence of the s-lvh class do you use in your style sheet?

Comment (amalisaah) [reply index.html:11]: ok ... you are using tailwind. Why?

File: src/components/Background.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+const Background = () => {
+  return (
+    <>
+      <div className="w-[497px] h-[497px] rounded-full blur-3xl bg-[#4400FF1F] absolute left-[-200px] top-[-200px]"></div>
+      <div className="h-screen w-screen bg-transparent">
+        <div
+          className=" min-w-[360px] w-[360px] h-[800px] border-white border-[12px] rounded-3xl overflow-hidden mx-auto mt-16"
+          style={{
+            background:
```
Comment (amalisaah) [inline src/components/Background.jsx]: why are you mixing inline styling with css classes from tailwind

File: src/components/Card.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const Card = () => {
```
Comment (amalisaah) [inline src/components/Card.jsx:1]: ???

File: src/components/Background.jsx (hunk 2 above)
Comment (bludojr) [reply src/components/Background.jsx]: > why are you mixing inline styling with css classes from tailwind

Thank you for the feedback.

I needed to apply a linear gradient, and at the time, I couldn’t find the appropriate utility in Tailwind to achieve the effect. To keep things moving, I used inline CSS for the gradient. I can explore Tailwind’s documentation further to find the correct utility or refactor it if needed.

Comment (amalisaah) [reply src/components/Background.jsx]: yh yh please do if we are going to be using tailwind then let do it right

File: index.html (hunk 1 above)
Comment (bludojr) [reply index.html:11]: Thank you for your feedback. I’ll refactor the code  for better organization and best practices.
As for using Tailwind CSS, I’ve gained a solid foundation in standard CSS, but I’m now exploring Tailwind for its efficiency, especially when paired with React.

File: src/components/Card.jsx (hunk 3 above)
Comment (bludojr) [reply src/components/Card.jsx:1]: Thank for the feedback!
The import React from 'react' is as a result of the RAFCE create function and export function shortcut for creating components
It adds import React from 'react' automatically

Comment (amalisaah) [reply src/components/Card.jsx:1]: if it is not needed delete it
and remember to use the formatter

**Review by amalisaah — APPROVED** [review]

**Pull request merged**